mobile.go

```diff
@@ -104,7 +105,10 @@ func doUpdateMobile(url string, out *os.File, updater Updater) error {
 	}
 
 	req.Header.Add("Accept-Encoding", "gzip")
-	pro.PrepareForFronting(req)
+
+	frontedURL := *req.URL
+	frontedURL.Host = "d2yl1zps97e5mx.cloudfront.net"
+	proxied.PrepareForFronting(req, frontedURL.String())
```

Comment:
Yeah, looks like ``ParallelPreferChained`` returns an error for any POST requests so we can't use it for updates:

See here: https://github.com/getlantern/flashlight/blob/devel/proxied/proxied.go#L179

Maybe we just use `ChainedThenFronted` then? Or we can continue to use the same http client as pro, that uses `ParallelPreferChained` only for GET/HEAD requests and `ChainedThenFronted` for others: I verified that works well already. But maybe move it into another package as we had before?